sdk/template/azure-template/CHANGELOG.md

```diff
@@ -26,6 +16,17 @@
 
 * Validate vcpkg publishing automation framework
 
+## 1.0.0-beta.4 (Unreleased)
+
+
```

Comment:
This is duplicated above, remove.
```suggestion
```

Reply:
Thank you for the suggestion, I have committed the changes.